pkg/auth/multiauthenticator_test.go

```diff
@@ -0,0 +1,135 @@
+package auth
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMultiAuthenticatorWithClientID_GetAuthenticator_Ok(t *testing.T) {
+	// Given.
+	jwtAuth, err := NewJwtAuthenticator(&JwtAuthConfig{
+		SharedSecret:  []byte("my-secret"),
+		UsernameClaim: UsernameClaimTypeSubject,
+	})
+	assert.NoError(t, err)
```

Comment:
did you mean to use "assert"? I have seen "require" usually since we want to fail right away.

Reply:
All the tests under this package were using assert, so continued to use that. I have no specific requirement for assert over require.